pyodide_build/pywasmcross.py

```diff
@@ -201,7 +207,15 @@ def handle_command(line, args, dryrun=False):
     emcc test.c
     ['emcc', 'test.c']
     """
-    # This is a special case to skip the compilation tests in numpy that aren't
+
+    if type(line) is dict:
```

Comment:
Could you add a comment explaining this a bit more?

Reply:
Nah. I'll remove it. The build log used to be a file of lists, but I changed it to a file of dictionaries. I allowed this function to handle both because I was too lazy to look for other ocurrences of writing to the log.